packages/deep-agent-chat/src/components/OptimizationWindow.tsx

```diff
@@ -400,65 +400,35 @@ export const OptimizationWindow = React.memo<OptimizationWindowProps>(
+      if (!displayMessages.length) return;
       optimizerMessagesEndRef.current?.scrollIntoView({ behavior: "smooth" });
-    }, [displayMessages, isExpanded]);
+    }, [displayMessages]);
 
     return (
       <>
         <div
           className={cn(
-            "flex flex-col overflow-hidden rounded-t-[10px] transition-all duration-300 ease-in-out",
-            isExpanded ? "h-[400px]" : "h-12",
+            "flex min-h-0 flex-1 flex-col overflow-hidden",
+            isExpanded ? "" : "",
           )}
         >
-          <div className="bg-primary flex min-h-12 items-center overflow-hidden rounded-t-xl border-none">
-            <Button
-              onClick={onToggle}
-              disabled={!optimizerClient}
-              className="flex h-full flex-1 cursor-pointer items-center justify-between bg-transparent px-4 text-sm font-medium text-white/80 transition-colors duration-200 ease-in-out hover:bg-transparent hover:text-white"
-            >
-              {optimizerClient
-                ? "Deep Agent Optimizer"
-                : "(Disabled) Deep Agent Optimizer"}
-            </Button>
-
-            {isExpanded && displayMessages.length > 0 && (
-              <Button
-                variant="ghost"
-                size="icon"
-                onClick={handleClear}
-                className="ml-auto bg-transparent text-white/80 transition-colors duration-200 hover:bg-transparent hover:text-white"
-              >
-                <RotateCcw size={16} />
-              </Button>
-            )}
-            <TooltipIconButton
-              onClick={onToggle}
-              disabled={!optimizerClient}
-              tooltip={
-                !optimizerClient
-                  ? "Set Optimizer Agent Environment Variables in FE Deployment"
-                  : "Expand Optimizer"
-              }
-              className="cursor-pointer bg-transparent px-6 text-white/80 transition-transform duration-200 ease-in-out hover:bg-transparent hover:text-white"
-            >
-              {isExpanded ? (
-                <X size={16} />
-              ) : (
-                optimizerClient && <Expand size={16} />
-              )}
-            </TooltipIconButton>
+          <div
+            className="flex items-center px-4 py-3"
+            style={{
+              fontFamily:
+                "Inter, ui-sans-serif, system-ui, -apple-system, 'Segoe UI', Roboto, Helvetica, Arial, 'Noto Sans', sans-serif",
+            }}
+          >
+            <span className="text-foreground text-md font-semibold tracking-wide">
+              AGENT CREATOR
+            </span>
           </div>
 
           <div
-            className={cn(
-              "flex min-h-0 flex-1 flex-col transition-opacity delay-100 duration-300 ease-in-out",
-              isExpanded ? "opacity-100" : "opacity-0",
-            )}
+            className={cn("flex min-h-0 flex-1 flex-col justify-between", "")}
```

Comment:
lots of small styling nits to cleanup
```suggestion
            className="flex min-h-0 flex-1 flex-col justify-between"
```